sktime/dists_kernels/compose_tab_to_panel.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""
+Composers that create panel pairwise transformers from table pairwise transformers
+"""
+
+__author__ = ["fkiraly"]
+
+import numpy as np
+
+from sktime.dists_kernels._base import BaseTrafoPwPanel
+
+
+class AggrDist(BaseTrafoPwPanel):
```

Comment:
```suggestion
class PairwiseAggrDist(BaseTrafoPwPanel):
```
I think Pairwise should be part of the name of any of these classes.

Reply:
well, but then @mloning would say, it should be at the end, to be in line with `sklearn` convention where things like `Classifier` etc come at the end. I don't like that since then the different scitypes don't come in blocks when ordering by alphabet, but I think it's better to conform with `sklearn` convention than inventing sth new.

Reply:
I like 'ordering by alphabet' as a criteria for how things should be named! Makes good sense.

But in this particular instance, I do not have strong feelings on whether Pairwise goes at the front or the end. I prefer front just because thats how its used in mathematical phrases

Reply:
>I like 'ordering by alphabet' as a criteria for how things should be named! Makes good sense.

... was this sarcasm or genuine? Hard to tell from just the writing.

Reply:
This was genuine! I already do this - naming things so that sorting by alphabet groups things sensibly -  in two or three large-ish folders on my laptop.